modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss

```diff
@@ -25,6 +25,10 @@
 		gap: 1rem;
 		width: 100%;
 
+		&-title {
+			min-width: 3.125rem;
```

Comment:
Hi @larissacribeiro, this change sure fixes the bug (now we can select the folder), but it creates another, visual one. See bellow. I think that adding the `text-truncate` class to the `span` element is a better fix. Have you tried that?
![screenshot-1698098961](https://github.com/liferay-objects/liferay-portal/assets/48384932/c099455e-b802-424f-9948-f7872cff8553)

Reply:
You're right, just did it!